src/app/components/report-problem/report-problem.component.html

```
@@ -0,0 +1,40 @@
+      @if (reportForm.get('details')?.hasError('required')) {
+        <mat-error>Por favor, forneça alguns detalhes.</mat-error>
+      }
+      @if (reportForm.get('details')?.hasError('maxlength')) {
+        <mat-error>Os detalhes não podem exceder 500 caracteres.</mat-error>
+      }
+    </mat-form-field>
+  </form>
+</mat-dialog-content>
+
+<mat-dialog-actions align="end">
+  <button mat-button (click)="onCancel()">Cancelar</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)